docs/SchemeFileFormat.md

```diff
@@ -65,6 +65,7 @@ Each query object instructs the tool to run a specific check on one or more sets
 |connectivityMap|Reports a summary of the allowed connections in the cluster|list of [config set](#configsets) names| 
 |expected|The expected sum of returned results by all sub-queries in this query (a warning is issued on mismatch)|integer|
 |expectedOutput|The file path of the expected output of this query (for connectivityMap or semanticDiff queries) |string|
+|expectedUnchecked|The number of input configs/config pairs that the query is not expected to be run on. Reasons for not checking the configs are listed [here](CmdLineQueriesResults.md#a-query-will-not-be-executed-when) |integer|
```

Comment:
Some mix in the terminology - sometime referring to "unchecked queries" and sometimes to "queries not executed".
Let's be consistent and replace "unchecked" with "not executed" (code + documentation).